utils/cardInfoUtil.ts

```diff
@@ -48,7 +48,7 @@ export class CardInfoUtil {
             startDate: new Date(result.startDate),
             endDate: new Date(result.endDate),
             weekday: result.weekday,
-            startTimeMinutes: totalMinutes(result.startDate),
+            startTimeMinutes: result.startDate ? totalMinutes(new Date(result.startDate)) : 0,
```

Comment:
Pretty minor but on line 48, you already have `new Date(result.startDate)`. Up to you if you want to reduce duplicate code.

Reply:
Also, in the cases when the ternary expression is false, what would `new Date(result.startDate)` return on line 48? Should there be a ternary expression there too then?

Reply:
true, I'll do new Date() in for false I guess.

Reply:
I think I'll just leave the one dup line in for now. I think it should be fine with how we're doing ternaries.